modules/cli/pages/cbsummary.adoc

```diff
@@ -0,0 +1,56 @@
+= cbsummary
+:page-topic-type: reference
+
+[abstract]
+The [.cmd]cbsummary tool connects to a set of Couchbase clusters and generates a summary report.
+
+== Description
+
+The [.cmd]cbsummary tool connects to a set of Couchbase clusters specified in a configuration file and generates a summary report. 
+
+The config file contains JSON specifying an array of information on each cluster, giving the Couchbase credentials to login, and one or more IP addresses for cluster nodes.
+
+The default report format includes RAM and core utilization across each specified cluster, since that information is useful in determining compliance with Couchbase licenses. If you specify --csv, then the report is generated in CSV instead of JSON. If, instead, you specify --full, then a much more detailed report is generated.
```

Comment:
```suggestion
The default report format includes RAM and core utilization across each specified cluster, since that information is useful in determining compliance with Couchbase licenses.
If you specify `--csv`, then the report is generated in CSV instead of JSON.
If, instead, you specify `--full`, then a much more detailed report is generated.
```